tools/topology/topology2/cavs/cavs-rt5682-rt1019.conf

```diff
@@ -0,0 +1,250 @@
+<include/common/vendor-token.conf>
+<include/common/tokens.conf>
+<include/common/manifest.conf>
+<include/common/pcm.conf>
+<include/common/pcm_caps.conf>
+<include/common/fe_dai.conf>
+<include/common/route.conf>
+<include/components/host.conf>
+<include/components/dai.conf>
+<include/components/pipeline.conf>
+<include/components/copier.conf>
+<include/dais/ssp.conf>
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+<include/common/data.conf>
+<include/dais/hw_config.conf>
+<include/pipelines/cavs/passthrough-playback.conf>
+<include/pipelines/cavs/passthrough-capture.conf>
+
+#
+# List of all DAIs
+#
+# PCM0 ----> volume -----> SSP1  (Speaker - ALC1019)
+# PCM1 <---> volume <----> SSP0  (Headset - ALC5682)
+# PCM2 <---- volume <---- DMIC01 (dmic 48k capture)
+Object.Dai {
+	SSP."0" {
+		# sof_rt5682.c in machine driver - id for speaker is 6 - 1 headset + 2 dmic + 3 hdmi (if not defined in quirks)
+		id 		6
+		direction	"duplex"
+		name		SSP0-Codec
+		default_hw_conf_id	0
+		sample_bits		24
+
+		Object.Base.hw_config."SSP0" {
+			id	0
+			mclk_freq	24000000
```

Comment:
is this real? we haven't use 24MHz since CNL?

Reply:
Made relevant changes, please review